fluid-project/trivet
# feat: replace Netlify CMS with Decap CMS

## Description

Work to replace netlify cms with decap cms. 

## Steps to test

1. Run npm install to get changed dependencies
2. Start the local server by running npm start
3. Go to /admin and see Decap cms log-in screen
4. Press sign in button and see cms page
5. Edit any posts and see the changes applied

## Related issues

Resolves #370
